backend/src/main/java/pl/zzpj/esportbetting/pandascore/MockedPandaScoreConnector.java

```diff
@@ -0,0 +1,201 @@
+package pl.zzpj.esportbetting.pandascore;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import pl.zzpj.esportbetting.interfaces.ESportRestApi;
+
+import java.io.IOException;
+
+public class MockedPandaScoreConnector implements ESportRestApi {
+    @Override
+    public JSONArray getAllLeagues() {
+        return null;
+    }
+
+    @Override
+    public JSONArray getAllSeries() {
+        return null;
+    }
+
+    @Override
+    public JSONArray getAllTournaments() {
+        return null;
+    }
+
+    @Override
+    public JSONArray getAllMatches() {
+        JSONArray jsonArray = new JSONArray();
+        try {
+            // not started
+            jsonArray.put(new JSONObject("{\n" +
+                    "\t\"live_embed_url\": null,\n" +
+                    "\t\"end_at\": null,\n" +
+                    "\t\"winner_id\": null,\n" +
+                    "\t\"begin_at\": \"2020-05-11T19:00:00Z\",\n" +
+                    "\t\"number_of_games\": 3,\n" +
+                    "\t\"id\": 560452,\n" +
+                    "\t\"modified_at\": \"2020-05-08T14:32:33Z\",\n" +
```

Comment:
Czy w Mocku potrzebne jest aż tyle danych skoro potem i tak parser wybiera jedynie część z nich?

Reply:
Zostawiłem więcej danych, bo to są takie dane, które potencjalnie mogą się przydać w przyszłości i być może będzie je można wykorzystać w projekcie